scripts/run_command_building.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+import subprocess
+import sys
+
+def find_binary(path, exe_identifier):
+    if sys.platform.startswith("win"):
+        suffix=".exe"
+    else:
+        suffix=""
+
+    binary_path = Path(path)
+    if binary_path.is_file():
+        return path.resolve()
+    if binary_path.is_dir():
+        results = []
+        for path_item in binary_path.iterdir():
+            if path_item.is_file() and (f"antares-{exe_identifier}{suffix}" == path_item.name):
+                results.append(path_item)
```

Comment:
break if the solver is found

Reply:
I kind of applied your suggestion, and shrunk the code a bit more